pkg/nodeps/wsl.go

```diff
@@ -0,0 +1,25 @@
+package nodeps
```

Comment:
```suggestion
package environment
```

Other environment functions should be moved too if you decide to go with a new package.

Reply:
Done.

Reply:
But it's still in nodeps, what have you done? I would move that file to the package. So in the end you should have 3 additional files in the environment package: wsl.go, gitpod.go and codespaces.go. Then we can call  e.g. `environment.IsWSL2()` which really looks nicer than `nodeps.IsWSL2()`. But I have to say this refactoring is also out of scope, so maybe we better do this cleanup in a dedicated patch anyway.